docs/configuring.rst

```diff
@@ -16,6 +16,7 @@ For example:
         'canonical_url': '',
         'analytics_id': 'UA-XXXXXXX-1',  #  Provided by Google in your dashboard
         'logo_only': False,
+        'logo_url': '',
```

Comment:
```suggestion
        'logo_href': '',
```

Motivation: Since Sphinx 4.0, `logo_url` is used by Sphinx as the path for the logo image file. So this will be confusing.